msteams-platform/m365-apps/prerequisites.md

```diff
@@ -71,14 +71,22 @@ For more information on Microsoft 365 release options, see [set up the Standard
 
 ### Desktop
 
-You can preview Teams apps running in Outlook on Windows desktop by using a recent *Current Channel (Preview)* build. Check if you have to [change the Microsoft 365 Apps update channel](/deployoffice/change-update-channels?WT.mc_id=M365-MVP-5002016) for your test tenant to install a Microsoft 365 Current Channel (Preview) build.
+You can preview Teams apps running in Outlook on Windows desktop by using a channel according to your testing scenario. Check if you have to [change the Microsoft 365 Apps update channel](/deployoffice/change-update-channels?WT.mc_id=M365-MVP-5002016) for your test tenant.
```

Comment:
The 2nd sentence here doesn't have much purpose except to provide a link for more info. Suggested rewrite:

You can preview Teams apps running in Outlook on Windows desktop by using a Microsoft 365 Apps update channel](/deployoffice/change-update-channels?WT.mc_id=M365-MVP-5002016) for your tenant according to the scenario you'd like to test.

Reply:
Updated.